gen/cmd/src/main.rs

```diff
@@ -205,12 +213,26 @@ fn main() {
         .unwrap_or_default()
         .collect();
     let suppress_system_headers = matches.is_present("suppress-system-headers");
+    let desired_number = matches
+        .value_of("generate-exact")
+        .map(|s| s.parse::<usize>().unwrap());
+    let header_counter = Cell::new(0);
```

Comment:
Nit: you could move this inside the 'if' block

Reply:
I don't think so? It needs to be scoped outside that so it can be used later.

Reply:
Ah yeah.